stories/PageHeader-stories.tsx

```diff
@@ -0,0 +1,120 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import type { Meta, StoryObj } from '@storybook/react-vite';
+import {
+  Button,
+  ButtonIcon,
+  ButtonVariety,
+  DropdownMenu,
+  IconArrowLeft,
+  IconEdit,
+  PageHeader,
+  PageHeaderProps,
+} from '../src';
+
+const meta: Meta<typeof PageHeader> = {
+  argTypes: {},
+  component: PageHeader,
+  title: 'Echoes/PageHeader',
+};
+
+export default meta;
+
+type Story = StoryObj<typeof PageHeader>;
+
+const render = (args: PageHeaderProps) => (
+  <div
+    style={{
+      border: '1px dotted blue',
+      display: 'flex',
+      flexDirection: 'column',
+      padding: '32px 24px',
+      width: '800px',
+    }}>
+    <PageHeader {...args} />
+  </div>
+);
+
+export const Full: Story = {
+  args: {
+    actions: (
+      <PageHeader.Actions>
+        <Button>Secondary action</Button>
+        <Button variety={ButtonVariety.Primary}>Primary action</Button>
+      </PageHeader.Actions>
+    ),
+    breadcrumbs: (
+      <PageHeader.Breadcrumbs
+        items={[
+          { linkElement: 'Breadcrumb item', to: 'https://sonarsource.com' },
+          { linkElement: 'Breadcrumb item', to: '' },
+        ]}
+      />
+    ),
+    children: (
+      <PageHeader.Navigation>
+        <PageHeader.NavigationItem to="/">Home</PageHeader.NavigationItem>
+        <PageHeader.NavigationItem to="/qp">Quality Profiles</PageHeader.NavigationItem>
+        <PageHeader.NavigationItem to="/rules">Rules</PageHeader.NavigationItem>
+
+        <PageHeader.NavigationDropdownItem
+          items={
+            <>
+              <DropdownMenu.ItemLink to="/3456">option 1</DropdownMenu.ItemLink>
+              <DropdownMenu.ItemLink to="/hiya">hiya</DropdownMenu.ItemLink>
+            </>
+          }>
+          More
+        </PageHeader.NavigationDropdownItem>
+      </PageHeader.Navigation>
+    ),
+    description: <PageHeader.Description>Page description</PageHeader.Description>,
+    metadata: <PageHeader.Metadata>Describe your metadata here</PageHeader.Metadata>,
+    title: (
+      <PageHeader.Title
+        prefix={
+          <ButtonIcon
+            Icon={IconArrowLeft}
+            ariaLabel="Go back"
+            variety={ButtonVariety.DefaultGhost}
+          />
+        }
+        suffix={
+          <ButtonIcon Icon={IconEdit} ariaLabel="Edit" variety={ButtonVariety.DefaultGhost} />
+        }>
+        Page title
+      </PageHeader.Title>
+    ),
+  },
+  parameters: {
+    controls: {
+      exclude: ['actions', 'breadcrumbs', 'children', 'description', 'metadata', 'title'],
+    },
+  },
```

Comment:
It's weird to have 0 controls on the story, maybe there is a better way to provide customization in the story? Using select type controls to toggle or not some of the children?